src/entities/BlockNode/BlockNode.spec.ts

```diff
@@ -1,34 +1,69 @@
-import { BlockNode } from './index';
-import { createBlockNodeName, createDataKey } from './types';
+import { BlockNode, createBlockNodeName, createDataKey } from './index';
 
-import { BlockTune, createBlockTuneName } from '../BlockTune';
-import { TextInlineNode } from '../inline-fragments/TextInlineNode';
+import { BlockTune, BlockTuneName } from '../BlockTune';
 import { ValueNode } from '../ValueNode';
 
 import type { EditorDocument } from '../EditorDocument';
 import type { BlockTuneConstructorParameters } from '../BlockTune/types';
-import type { TextInlineNodeConstructorParameters } from '../inline-fragments/TextInlineNode';
 import type { ValueNodeConstructorParameters } from '../ValueNode';
+import { RootInlineNode } from '../inline-fragments';
+import { describe } from '@jest/globals';
+
+jest.mock('../BlockTune', () => ({
+  BlockTune: jest.fn().mockImplementation(() => ({
+    get serialized() {
+      return {};
+    },
+    update() {
+      return;
+    },
+  }) as unknown as BlockTune),
+}));
+
+jest.mock('../inline-fragments', () => ({
+  RootInlineNode: jest.fn().mockImplementation(() => ({
+    get serialized() {
+      return {};
+    },
+  }) as RootInlineNode),
+}));
+
+jest.mock('../ValueNode', () => ({
+  ValueNode: (class {
```

Comment:
Why don't you use jest.fn() here?

Reply:
It's better to mock classes with classes I believe, I'll also move this to a mock file

Reply:
Looks much better now